packages/node/test/e2e/cli-util.js

```diff
@@ -20,45 +20,89 @@ export const CLI_PATH = fileURLToPath(
 
 /**
  * @import {ExecFileException} from 'node:child_process'
- * @import {RunCLIOutput} from '../types.js'
+ * @import {ExitCode, RunCLIOptions, RunCLIOutput} from '../types.js'
  * @import {ExecutionContext} from 'ava'
  */
 
 /**
  * Run the `@lavamoat/node` CLI with the provided arguments
  *
  * @template [Context=unknown] Default is `unknown`
+ * @overload
+ * @param {string[]} args CLI arguments
+ * @param {RunCLIOptions} [options] Options
+ * @returns {Promise<RunCLIOutput>}
+ */
+
+/**
+ * Run the `@lavamoat/node` CLI with the provided arguments
+ *
+ * @template [Context=unknown] Default is `unknown`
+ * @overload
  * @param {string[]} args CLI arguments
  * @param {ExecutionContext<Context>} [t] AVA test context
+ * @param {RunCLIOptions} [options] Options
  * @returns {Promise<RunCLIOutput>}
  */
 
-export const runCLI = async (args, t) => {
+/**
+ * Run the `@lavamoat/node` CLI with the provided arguments
+ *
+ * @template [Context=unknown] Default is `unknown`
+ * @param {string[]} args CLI arguments
+ * @param {ExecutionContext<Context> | RunCLIOptions} [tOrOptions] AVA test
+ *   context
+ * @param {RunCLIOptions} [options] Options
+ * @returns {Promise<RunCLIOutput>}
+ */
+
+export const runCLI = async (
+  args,
+  tOrOptions,
+  { cwd = process.cwd() } = {}
+) => {
   await Promise.resolve()
 
+  /** @type {ExecutionContext<Context> | undefined} */
+  let t
+  if (tOrOptions) {
+    if ('plan' in tOrOptions) {
+      t = tOrOptions
+    } else {
+      cwd = tOrOptions?.cwd ?? cwd
+    }
+  }
+
   /** @type {string} */
   let stdout
   /** @type {string} */
   let stderr
-  /** @type {ExecFileException['code']} */
+  /** @type {ExitCode} */
   let code
 
   /**
-   * Full command; for display purposes only.
+   * Generate the full command for logging (and snapshots!?)
```

Comment:
do we still want to mention `(and snapshots!?)`

Reply:
hopefully it's weird enough for readers to want to continue reading the explanation below 😄

Reply:
relative planetary positions 🪐 lol